solidity/depool/README.md

```diff
@@ -930,6 +1031,61 @@ Result: {
 }
 ```
 
+The round parameters displayed by the get-method are the following:
+
+`completionReason`: the code for the reason the round was completed. Remains "0x0", until round actually is completed. The possible reasons are:
+
```

Comment:
I think it would be better to add 0  - Round is not completed yet 
because that's also a possible  value

Reply:
added